core/filter_api.php

```diff
@@ -3778,6 +3742,41 @@ function filter_get( $p_filter_id, array $p_default = null ) {
 	return $t_filter;
 }
 
+/**
+ * Return a standard filter
+ * @param string $p_filter_name     The name of the filter
+ * @param integer|null $p_user_id   A user id to build this filter. Null for current user
+ * @return null|boolean|array       null filter not found, false invalid filter, otherwise the filter.
+ */
+function filter_standard_get( $p_filter_name, $p_user_id = null ) {
```

Comment:
How about having parameters as: filter name, project id, user id?  With the last two defaulted to null.

Reply:
That's ok. Currently project_id is not used from the caller. Can you extend this as needed after this PR?

Reply:
sounds good